sync/sync_head.go

```diff
@@ -102,6 +102,7 @@ func (s *Syncer[H]) subjectiveHead(ctx context.Context) (H, error) {
 		log.Warnw("subjective initialization with an old header", "height", trustHead.Height())
 	}
 	log.Warn("trusted peer is out of sync")
+	s.metrics.recordTrustedPeersOutOfSync(s.ctx)
```

Comment:
Pls record some further details about it (trustHead and how "old" it is)

Reply:
it's height, timestamp, how "outdated", etc.

Reply:
Do you mean recording it as attributes? It is impossible to record values like height, timestamp and drift from head using metric attributes directly, as it will cause major cardinality issues in timeseries db. Some like drift are measured indirectly by difference between network head and sync head. 
If you meant something else, could you please elaborate?

Reply:
Would it still cause cardinality issues if this event is rare? @walldiss

Reply:
We should at least update the logging there to include following information

Reply:
The switch statement above already logs all necessary information.